src/snippets/template.py2.tmpl

```diff
@@ -5,7 +5,7 @@ import sys, errno, re, json, ssl
 import urllib2
 from time import sleep
 
-BASE_URL = "<%= url %>?page_size=200<%= fileType === 'fasta' ? '&extra_fields=sequence' : '' %>"
+BASE_URL = "<%= url %>?page_size=100<%= fileType === 'fasta' ? '&extra_fields=sequence' : '' %>"
```

Comment:
Why are you reducing it to 100?

Reply:
Probably just a leftover from testing, I might have been testing with 10 and just added '0' to my test.